charts/kube-prometheus-stack/values.yaml

```diff
@@ -679,6 +679,8 @@ grafana:
       ##
       annotations: {}
       multicluster: false
+      etcd:
+        multicluster: false
```

Comment:
Hey @invidian - could you please change this to be under a single `multicluster` and just have then `multicluster.global` and `multicluster.etcd`?

Reply:
Sure, should this be a breaking change then or should I do some hackery with `printf` to make it backwards compatible (as `multicluster` key type will change)?

Reply:
@Let's make it a breaking change without hackeries. The minor version of the chart will need to be bumped though.

Reply:
Done. I bumped minor version, not major, so I didn't add any changelog to README.md.